src/components/MyAccount/NewSettings/PhoneEmailForm.tsx

```diff
@@ -0,0 +1,305 @@
+                      invalidText={formUtils.errorMessage}
+                      onChange={(e) => handleInputChange(e, index)}
+                      isRequired
+                      isClearable
+                      isClearableCallback={() => handleClearableCallback(index)}
+                    />
+                    {(!isEmail || index !== 0) && (
+                      <Button
+                        aria-label={`Remove ${formUtils.inputLabel.toLowerCase()}`}
+                        buttonType="text"
+                        id="remove-input-btn"
+                        onClick={() => handleRemove(index)}
+                      >
+                        {""}
+                        <Icon name="actionDelete" size="large" />
+                      </Button>
+                    )}
+                  </Flex>
+                </Form>
+              ))}
+              <Button
+                id={`add-${inputType}-button`}
+                buttonType="text"
+                onClick={handleAdd}
+                size="large"
+                sx={{
+                  justifyContent: "flex-start",
+                  width: { base: "87%", md: "300px" },
+                  padding: "xxs",
+                }}
+              >
+                {formUtils.addButtonLabel}
+              </Button>
+            </Flex>
+          ) : isEmail || tempInputs.length != 0 ? (
+            <Flex
+              marginLeft={{ base: "l", lg: "unset" }}
+              flexDir="row"
+              alignItems="flex-start"
+            >
+              <Flex flexDir="column" alignItems="flex-start">
+                {tempInputs.map((input, index) => (
+                  <Text
+                    key={index}
+                    sx={{ paddingTop: "xs", marginBottom: "xs" }}
+                  >
+                    {input}{" "}
+                    {index === 0 && inputs.length > 1 && (
+                      <span
+                        style={{
+                          display: "inline",
+                          padding: 0,
+                          margin: 0,
+                          color: "var(--nypl-colors-ui-gray-semi-dark)",
+                        }}
+                      >
+                        (P)
+                      </span>
+                    )}
+                  </Text>
+                ))}
+              </Flex>
+              <Button
+                id="edit-input-button"
+                buttonType="text"
+                onClick={() => {
+                  setTempInputs([...inputs])
+                  setIsEditing(true)
+                }}
+                sx={{
+                  paddingLeft: "xs",
+                  paddingRight: "xs",
+                  marginLeft: "xxl",
+                }}
+              >
+                <Icon name="editorMode" align="left" size="medium" />
+                Edit
+              </Button>
+            </Flex>
+          ) : (
+            <Button
```

Comment:
This is the same button as what's on lines 214-226 except that the `onClick` here has an extra function. Try to extract this because you'd have to keep track of both in case the style changes in either one of them.